modules/subscriptions.php

```diff
@@ -616,7 +616,7 @@ function widget( $args, $instance ) {
 					<p class="error"><?php esc_html_e( 'You have already subscribed to this site. Please check your inbox.', 'jetpack' ); ?></p>
 				<?php break;
 				case 'success' :
-					echo wpautop( $success_message );
+					echo wpautop( str_replace( '%d', number_format_i18n( $subscribers_total['value'] ), $success_message ) );
```

Comment:
For consistency I would remove the `echo` and just use `?> <?php` tags

You could wrap the `wpautop( )` function in a div with a class `subscribe-info` for example.
Which would allow users to style it. Another way of doing this would be to provide the same feature is. passing the whole string though the `apply_filters()` function. This would allow the user change the text to their will. 

I am also not sure about the string replace used here. Since we would also match anything string containing `%d` (maybe even some urls in links would be broken ). I think something like a shortcode format string would work better `[total-subscribers]`.